buildout.cfg

```diff
@@ -29,15 +26,37 @@ tslib =
 # Note that lxml requires libxml2-dev and libxslt-dev to be
 # present if you want to build it from source.
 lxml  =
+pandas = 0.19.2
+zc.buildout = 2.9.4
```

Comment:
Liever unpinnen (ik heb 2.11.2, en dit conflicteert hiermee, ik kom in een infinite loop terecht)